test/TwilioVoiceReactNativeExample/e2e/basic.test.ts

```diff
@@ -0,0 +1,120 @@
+import { device, element, expect as detoxExpect, by, waitFor } from 'detox';
+import { expect as jestExpect } from 'expect';
+import twilio from 'twilio';
+
+const DEFAULT_TIMEOUT = 10000;
+
+const bootstrap = () => {
+  const accountSid = process.env.ACCOUNT_SID;
+  const authToken = process.env.AUTH_TOKEN;
+  const mockClientId = process.env.CLIENT_IDENTITY;
+
+  if (
+    [accountSid, authToken, mockClientId].some((v) => typeof v !== 'string')
+  ) {
+    throw new Error('Missing env var.');
+  }
+
+  const twilioClient = twilio(accountSid, authToken);
+
+  return { twilioClient, clientId: mockClientId as string };
+};
+
+describe('basic', () => {
```

Comment:
We should do proper naming of test suites. "basic" test suite does not tell us what's included. Can we rename this?